src/blueprint/azext_blueprint/custom.py

```diff
@@ -228,17 +406,21 @@ def add_blueprint_resource_group(client,
                                  description=None,
                                  depends_on=None,
                                  tags=None):
-    body = client.get(resource_scope=resource_scope, blueprint_name=blueprint_name).as_dict()
+    args = {
+        "name": blueprint_name,
+        "resource_scope": resource_scope,
+    }
+    body = BlueprintShow(cli_ctx=cmd.cli_ctx)(args)

```

Comment:
Please add `command_args=args` otherwise the coverage tool will not detect this command as code gen command.

```suggestion
    body = BlueprintShow(cli_ctx=cmd.cli_ctx)(command_args=args)

```